runtime/vm/CRIUHelpers.cpp

```diff
@@ -538,6 +596,8 @@ initializeCriuHooks(J9VMThread *currentThread)
 		addInternalJVMCheckpointHook(currentThread, TRUE, NULL, FALSE, criuRestoreInitializeTrace);
 		addInternalJVMCheckpointHook(currentThread, TRUE, NULL, FALSE, criuRestoreInitializeXrs);
 		addInternalJVMCheckpointHook(currentThread, TRUE, NULL, FALSE, criuRestoreInitializeDump);
+		addInternalJVMCheckpointHook(currentThread, TRUE, NULL, FALSE, criuRestoreCheckForJdwp);
+		addInternalJVMCheckpointHook(currentThread, TRUE, NULL, FALSE, criuRestoreCheckForDebugInterpreterRequest);
```

Comment:
`criuRestoreCheckForJdwp()` and `criuRestoreCheckForDebugInterpreterRequest()` don't need to be run as hook methods.
I think the code setting `J9VM_CRIU_TRANSITION_TO_DEBUG_INTERPRETER` can be in `checkTransitionToDebugInterpreter()` before `isTransitionToDebugInterpreterRequested()`.